server/src/services/desks.service.ts

```diff
@@ -53,6 +59,59 @@ export function getDesksService({
   ];
 }
 
+export async function getDesksServiceReal({
+  zoneId,
+  deskIds = [],
+  count = 10,
+  unit = "days",
+}: GetDesksServiceParams): Promise<DeskDto[]> {
+  const desksResult: DeskDto[] = [];
+
+  if (Array.isArray(deskIds) && deskIds.length > 0) {
```

Comment:
tady neni potreba kontrolovat, ze je to array kdyz mu defaultne davas prazdny list.

Reply:
Asi si to tam radsi necham 😅